samtranslator/swagger/swagger.py

```diff
@@ -788,6 +789,37 @@ def add_models(self, models):
 
             self.definitions[model_name.lower()] = schema
 
+    def add_resource_policy(self, resource_policy):
+        """
+        Add resource policy definition to Swagger.
+
+        :param dict resource_policy: Dictionary of resource_policy statements which gets translated
+        :return:
+        """
+        if resource_policy is None:
+            return
+
+        custom_statements = resource_policy.get('CustomStatements')
+
+        if custom_statements is not None:
+            if not isinstance(custom_statements, list):
+                raise ValueError("Invalid input. CustomStatements must be a list.")
```

Comment:
Did we want to enforce this? I think the swagger property requires a list (can't tell for sure though, since they don't document it), but it might be nice if we were to wrap this into a list instead of fail.

If we are going to fail here, could you add an error case test to make sure that we catch the ValueError and don't just throw a 5xx error that gets returned as an "Internal server error" to the user?

Reply:
I changed it to wrap into a list instead of throwing an error.